CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+#How to Contribute
+Below are the instructions for making changes to simple chat. There are a few important guidlines we'd wish for you to follow.
+These will make it easier for you to contribute and for us to follow. Thank you for contributing!
+
+##Getting Started
+If you found a bug there are two options. If you have a solution, please follow these steps:
+-Make sure you have a [Github account](https://github.com/join)
+-Fork the repository
```

Comment:
Since this is a step-by-step guide, why don't make it a numbered list like this:
`1. [Fork](https://github.com/mbrandau/simple-chat/fork) the repository`
1. [Fork](https://github.com/mbrandau/simple-chat/fork) the repository